test/service/test_repository_projects.py

```diff
@@ -53,7 +53,6 @@ def assert_file_type_summaries(hit):
             assert_file_type_summaries(hit)
             self.assertTrue('projects' in hit)
             self.assertTrue('specimens' in hit)
-            self.assertTrue('projectSummary' in hit)
```

Comment:
Why not make assertFalse instead, to test that it is truly removed?

Reply:
there is also `assertIn` which would yield a better failure message.

Reply:
And yes, inverting rather than removing makes sense. So use `assertNotIn`

Reply:
Implemented check for non-existence of the 'projectSummary' object with `assertNotIn`